README.md

```diff
@@ -235,6 +235,15 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Arguments
+
+Execute with the config file on the command line using -c
+
+The default locations are ```/etc/pump.io.json``` and ```~/.pump.io.json```.
+
+    ./bin/pump -c pump.io.json
```

Comment:
Since people should probably be installing from npm, change `./bin/pump` to `pump`.